src/util/auctions.ts

```diff
@@ -0,0 +1,166 @@
+import { ethers } from "ethers";
```

Comment:
@Remscar So to confirm, after speaking today and clearing some of this up we're comfortable leaving things where they are for now instead of abstracting out into an `actions` or similarly named folder?

Reply:
Yes, leaving them as is would be great.